yandex/internal/yq/sdk/sdk.go

```diff
@@ -0,0 +1,79 @@
+package sdk
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+
+	"github.com/yandex-cloud/terraform-provider-yandex/yandex/internal/yq/sdk/client"
+	"github.com/yandex-cloud/terraform-provider-yandex/yandex/internal/yq/sdk/client/connection"
+	"github.com/yandex-cloud/terraform-provider-yandex/yandex/internal/yq/sdk/object_storage_connection"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// SDK is Yandex Query SDK
+type SDK struct {
+	conf       Config
+	connDialer connection.ConnectionDialer
+}
+
+func validateConfig(conf *Config) error {
+	if conf.FolderID == "" {
+		return fmt.Errorf("\"folder_id\" is required to YQ SDK")
+	}
+
+	if conf.AuthToken == "" {
+		return fmt.Errorf("\"token\" is required to YQ SDK")
+	}
+
+	if conf.Endpoint == "" {
+		return fmt.Errorf("\"yq_endpoint\" is required to YQ SDK")
```

Comment:
`to` -> `for`

Reply:
> `to` -> `for`

@lifthelm Maybe you want to change it in code?